minetest-database.h

```diff
@@ -31,12 +31,12 @@ with this program; if not, write to the Free Software Foundation, Inc.,
  * It's a lot more complicated than it looks.
  */
 
-static inline int unsigned_to_signed(unsigned i, unsigned max_positive)
+static inline int unsigned_to_signed(int64_t i, unsigned max_positive)
```

Comment:
This doesn't seem logical. It makes one wonder why this parameter had to be `int64_t`. Apparently, because `pythonmodulo` returns `int64_t`...

I think a better fix would be to make pythonmodulo return `unsigned`. At the same time, it probably makes sense to change the second parameter of `pythonmodulo` from `int16_t` to `unsigned` as well. In any case, `pythonmodulo` should return the same type as its second parameter, whether it's `int16_t` or `unsigned`.